utils/bazel/llvm-project-overlay/llvm/BUILD.bazel

```diff
@@ -32,6 +32,7 @@ exports_files([
     "include/llvm/IR/Intrinsics.td",
     "include/llvm/Option/OptParser.td",
     "utils/lit/lit.py",
+    "utils/lldbDataFormatters.py",
```

Comment:
**Suggestion** Would it make sense to add a comment here confirming that this has usage by downstream users of LLVM?

For context for other reviewers, this was originally added a few months ago in https://github.com/llvm/llvm-project/commit/b1a93dbaba87119556c1138b64cde6aa5ee8a854, but then removed a few days ago in https://github.com/llvm/llvm-project/commit/bb9ff32867d633787b5dd2572bc6e2fe5dade107 because it appeared unused within LLVM. 

To hedge against a similar cleanup effort removing these in the future, a comment in the source confirming this has external usage might help.

Reply:
Yes, that was exactly my comment in https://github.com/llvm/llvm-project/pull/114620#issuecomment-2457924669.

Reply:
Adding a comment here https://github.com/llvm/llvm-project/pull/115556